demo/locales/en.json

```diff
@@ -113,7 +113,7 @@
   "DEMO_LCEVC_SECTION_HEADER": "MPEG-5 Part-2 LCEVC",
   "DEMO_LCEVC_DRAW_LOGO": "MPEG-5 Part-2 LCEVC Show/Hide Logo",
   "DEMO_LCEVC_DYNAMIC_PERFORMANCE_SCALING": "MPEG-5 Part-2 LCEVC - Dynamic Performance Scaling",
-  "DEMO_LCEVC_LOG_LEVEL": "MPEG-5 Part-2 LCEVC Log Levels",
+  "DEMO_LCEVC_LOG_LEVEL": "MPEG-5 Part-2 LCEVC Log Level",
```

Comment:
These don't match the source.json file.  I think the source.json ones look more correct to me.  I tried to edit and fix it myself, but you have not given permission for maintainer edits.

Reply:
Updated these to match the source.json.